filebeat/harvester/reader/docker_json_test.go

```diff
@@ -32,12 +32,28 @@ func TestDockerJSON(t *testing.T) {
 			stream:        "all",
 			expectedError: true,
 		},
+		// Wrong CRI
+		{
+			input:         [][]byte{[]byte(`{this is not JSON nor CRI`)},
```

Comment:
I'm guessing `2017-09-12T22:32:21.212861448Z stdout` as input would cause a panic, it would be nice to add it to the list of tests :)

Reply:
Good catch! I added it to the test cases, its covered by checking array len.